src/Roundabout/roundabout-label.tsx

```diff
@@ -2,8 +2,8 @@
 
 export function RoundaboutLabel({ value }: { value: string }) {
     return (
-        <div className="fr-col-12">
-            <h1 className="roundabout-label">{value}</h1>
+        <div className="fr-col-12" style={{ borderBottom: "solid #DDDDDD 1px" }}>
```

Comment:
use color from dsfr

Reply:
maybe pass fr-border and in the inline style borderTop : 0 borderLeft: 0 borderRight:0 ...

Reply:
or try <hr> with the class fr-border maybe

Reply:
hr

Reply:
The `<hr />` seemed to be the simplest solution